<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="SpectrumTextEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>71</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>711</width>
      <height>441</height>
     </rect>
    </property>
    <widget class="QLabel" name="SummLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>SummLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="ScalersTextEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ScalersLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scalers:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spectrum:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalRecordLabel">
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Records:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>52</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="TotalRecordNumberEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>128</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="PrevRecordPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="RecordNumberEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextRecordPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="RewriteButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>440</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionAddRecord"/>
   <addaction name="actionDeleteRecord"/>
   <addaction name="actionSaveToTXT"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icons/blue-folder-horizontal-open.png</normaloff>:/images/Icons/blue-folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icons/blue-document.png</normaloff>:/images/Icons/blue-document.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create New File</string>
   </property>
  </action>
  <action name="actionAddRecord">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icons/plus.png</normaloff>:/images/Icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Record After</string>
   </property>
   <property name="toolTip">
    <string>Add Record After</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icons/disk-black.png</normaloff>:/images/Icons/disk-black.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDeleteRecord">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icons/minus.png</normaloff>:/images/Icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete Current Record</string>
   </property>
   <property name="toolTip">
    <string>Delete Current Record</string>
   </property>
  </action>
  <action name="actionSaveToTXT">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icons/receipt-text.png</normaloff>:/images/Icons/receipt-text.png</iconset>
   </property>
   <property name="text">
    <string>SaveToTXT</string>
   </property>
   <property name="toolTip">
    <string>Save Current Record to TXT File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
